src/concrete/ml/onnx/convert.py

```diff
@@ -208,7 +221,62 @@ def get_equivalent_numpy_forward_from_onnx(
             f"Available ONNX operators: {', '.join(sorted(IMPLEMENTED_ONNX_OPS))}"
         )
 
+    return equivalent_onnx_model
+
+
+def get_equivalent_numpy_forward_from_onnx(
+    onnx_model: onnx.ModelProto,
+    check_model: bool = True,
+) -> Tuple[Callable[..., Tuple[numpy.ndarray, ...]], onnx.ModelProto]:
+    """Get the numpy equivalent forward of the provided ONNX model.
+
+    Args:
+        onnx_model (onnx.ModelProto): the ONNX model for which to get the equivalent numpy
+            forward.
+        check_model (bool): set to True to run the onnx checker on the model.
+            Defaults to True.
+
+    Returns:
+        Callable[..., Tuple[numpy.ndarray, ...]]: The function that will execute
+            the equivalent numpy function.
+    """
+
+    equivalent_onnx_model = preprocess_onnx_model(onnx_model, check_model)
+
     # Return lambda of numpy equivalent of onnx execution
     return (
         lambda *args: execute_onnx_with_numpy(equivalent_onnx_model.graph, *args)
     ), equivalent_onnx_model
+
+
+def get_equivalent_numpy_forward_from_onnx_tree(
+    onnx_model: onnx.ModelProto,
+    check_model: bool = True,
+    lsbs_to_remove_for_trees: Optional[Tuple[int, int]] = None,
+) -> Tuple[Callable[..., Tuple[numpy.ndarray, ...]], onnx.ModelProto]:
+    """Get the numpy equivalent forward of the provided ONNX model.
```

Comment:
very small detail but maybe specify that this is for tree-based models only in the docstring as well